hasura/graphql-engine
# added YugabyteDB to the core databases in docs

### Description
This adds Yugabyte to the core databases in the Hasura documentation. It follows a very similar pattern to the Citus documentation.

### Changelog


### Affected components


### Related Issues
N/A

### Solution and Design
This adds Yugabyte to the core databases in the Hasura documentation. It follows a very similar pattern to the Citus documentation.

### Steps to test and verify
* Run `make livehtml` in `docs` folder
* Navigate to Core > Databases. See that YugabyteDB is included.

### Limitations, known bugs & workarounds
N/A

### Server checklist
N/A

#### Catalog upgrade

Does this PR change Hasura Catalog version?

#### Metadata


Does this PR add a new Metadata feature?
  - Does `run_sql` auto manages the new metadata through schema diffing?
  - Does `run_sql` auto manages the definitions of metadata on renaming?
  - Does `export_metadata`/`replace_metadata` supports the new metadata added?


#### GraphQL
   
   

#### Breaking changes


  1. Metadata API

     Existing `query` types:
     

  2. GraphQL API

     Schema Generation:
     
     
     

     Schema Resolve:-
     
     

  3. Logging
